tailwind.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./packages/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
no `jsx?` or `mdx?` pls 🙏🏼

Reply:
Made in https://github.com/podcodar/webapp/pull/173/commits/702b3cfa0b0afb78a5599a4bbfe9b179efc95ecd